src/cascadia/TerminalApp/SettingsTab.cpp

```diff
@@ -0,0 +1,177 @@
+#include "ActionAndArgs.h"
+#include "ActionArgs.h"
+
+using namespace winrt;
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Microsoft::Terminal::TerminalControl;
+using namespace winrt::Windows::System;
+
+namespace winrt
+{
+    namespace MUX = Microsoft::UI::Xaml;
+    namespace WUX = Windows::UI::Xaml;
+}
+
+namespace winrt::TerminalApp::implementation
+{
+    SettingsTab::SettingsTab()
+    {
+        Content(winrt::Microsoft::Terminal::Settings::Editor::MainPage());
+
+        _MakeTabViewItem();
+        _MakeSwitchToTabCommand();
+        _CreateContextMenu();
+        _CreateIcon();
+    }
+
+    // Method Description:
+    // - Initializes a TabViewItem for this Tab instance.
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - <none>
+    void SettingsTab::_MakeTabViewItem()
+    {
+        TabViewItem(::winrt::MUX::Controls::TabViewItem{});
+        TabViewItem().Header(winrt::box_value(Title()));
+    }
+
+    // Method Description:
+    // - Focus the settings UI
+    // Arguments:
+    // - focusState: The FocusState mode by which focus is to be obtained.
+    // Return Value:
+    // - <none>
+    void SettingsTab::Focus(WUX::FocusState focusState)
+    {
+        _focusState = focusState;
+
+        if (_focusState != FocusState::Unfocused)
+        {
+            Content().Focus(focusState);
+        }
+    }
+
+    WUX::FocusState SettingsTab::FocusState() const noexcept
+    {
+        return _focusState;
+    }
+
+    // Method Description:
+    // - Set the icon on the TabViewItem for this tab.
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - <none>
+    winrt::fire_and_forget SettingsTab::_CreateIcon()
+    {
+        auto weakThis{ get_weak() };
+
+        co_await winrt::resume_foreground(TabViewItem().Dispatcher());
+
+        if (auto tab{ weakThis.get() })
+        {
+            auto fontFamily = winrt::WUX::Media::FontFamily(L"Segoe MDL2 Assets");
+            auto glyph = L"\xE713"; // This is the Setting icon (looks like a gear)
+
+            // The TabViewItem Icon needs MUX while the IconSourceElement in the CommandPalette needs WUX...
+            IconSource(GetFontIcon<winrt::WUX::Controls::IconSource>(fontFamily, 12, glyph));
+            TabViewItem().IconSource(GetFontIcon<winrt::MUX::Controls::IconSource>(fontFamily, 12, glyph));
+
+            // Update SwitchToTab command's icon
+            SwitchToTabCommand().IconSource(GetFontIcon<winrt::WUX::Controls::IconSource>(fontFamily, 12, glyph));
+        }
+    }
+
+    // Method Description:
+    // - Gets the title string of the settings UI
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - the title string of the last focused terminal control in our tree.
+    winrt::hstring SettingsTab::GetActiveTitle() const
+    {
+        // TODO: This _could_ change depending on what page of the settings UI is open/focused?
+        return Title();

```

Comment:
Good thinking. This should probably be left as polish for after we get this into master though. Let's remove the comment for now and just add a note in the Epic that we should do this later.